discovery-provider/src/solana/solana_program_indexer.py

```diff
@@ -169,6 +169,9 @@ async def process_txs_batch(self, tx_sig_batch_records: List[str]):
         for tx_sig in tx_sig_batch_records:
             parsed_transactions.append(tx_sig_futures_map[tx_sig])
 
+        # Reverse parsed transactions so oldest is first
+        parsed_transactions.reverse()
```

Comment:
i was actually thinking of doing this in my last [PR](https://github.com/AudiusProject/audius-protocol/pull/2983) but decided to keep the ordering the same as we do elsewhere. but if we really do want oldest first, then maybe it'd be good to just reverse ordering in get_transaction_batches_to_process.

Reply:
https://linear.app/audius/issue/PLAT-128/minor-follow-ups-to-indexing-admin-user-functions-pr